test/integration.js

```diff
@@ -352,7 +351,7 @@ beforeEach(function () {
     })
 
     describe('source-maps', () => {
-      beforeEach(cb => rimraf('tmp/source-map', cb))
+      beforeEach(cb => rmSync('tmp/source-map', { recursive: true, ...cb }))
```

Comment:
Same as above. `force: true` is also needed I think.

```suggestion
      beforeEach(() => rmSync('tmp/source-map', { force: true, recursive: true }))
```

Reply:
should be using `rm` rather than `rmSync` and passing `cb` as last parameter. fixed.